packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -112,7 +112,8 @@ abstract class RunCommandBase extends FlutterCommand with DeviceBasedDevelopment
       ..addFlag('trace-systrace',
         negatable: false,
         help: 'Enable tracing to the system tracer. This is only useful on '
-              'platforms where such a tracer is available (Android and Fuchsia).',
+              'platforms where such a tracer is available (Android, iOS, '
+              'macOS and Fuchsia).',
```

Comment:
Is it set up in the engine for macOS?

Reply:
Dart supports it, I didn't verify that the engine is piping it down to dart correctly though.  I'm not too familiar with the macos flow.

Reply:
So should this have been changed to document that it works on macOS?

Reply:
Yea, macOS is a platform "where such a tracer is available".